lib/srv/reexec.go

```diff
@@ -169,7 +169,9 @@ func RunCommand() (io.Writer, int, error) {
 			return errorWriter, teleport.RemoteCommandFailure, trace.BadParameter("failed to resolve tty soft link: %v", err)
 		}
 		err := createUaccSession(ttyName, &c)
-		if err != nil {
+		if err != nil && !trace.IsAccessDenied(err) {
+			return errorWriter, teleport.RemoteCommandFailure, trace.Wrap(err)
+		}
 			if !trace.IsAccessDenied(err) {
```

Comment:
If you auto-applied the suggested changes - it's my bad (or GH's) that now the old handling is still there and it is redundant. Here and below.

Reply:
I did, just pushed a fix.